lib/VCMIDirs.cpp

```diff
@@ -535,23 +545,50 @@ std::string VCMIDirsXDG::libraryName(const std::string& basename) const { return
 #ifdef VCMI_ANDROID
 class VCMIDirsAndroid : public VCMIDirsXDG
 {
+	std::string basePath;
+	std::string internalPath;
+	std::string nativePath;
 public:
+	boost::filesystem::path fullLibraryPath(const std::string &desiredFolder, const std::string &baseLibName) const override;
+	boost::filesystem::path libraryPath() const override;
 	boost::filesystem::path userDataPath() const override;
 	boost::filesystem::path userCachePath() const override;
 	boost::filesystem::path userConfigPath() const override;
 
 	std::vector<boost::filesystem::path> dataPaths() const override;
+
+	void init() override;
 };
 
-// on Android HOME will be set to something like /sdcard/data/Android/is.xyz.vcmi/files/
-bfs::path VCMIDirsAndroid::userDataPath() const { return getenv("HOME"); }
+bfs::path VCMIDirsAndroid::libraryPath() const { return nativePath; }
+bfs::path VCMIDirsAndroid::userDataPath() const { return basePath; }
 bfs::path VCMIDirsAndroid::userCachePath() const { return userDataPath() / "cache"; }
 bfs::path VCMIDirsAndroid::userConfigPath() const { return userDataPath() / "config"; }
 
+boost::filesystem::path VCMIDirsAndroid::fullLibraryPath(const std::string &desiredFolder,
+														 const std::string &baseLibName) const {
```

Comment:
How do you suggest to align the arguments in this case? According to wiki "(...) please align your arguments to match the opening brace". Should I use normal continuation indent instead?